plugins/statistics.koplugin/main.lua

```diff
@@ -1942,6 +1942,7 @@ function ReaderStatistics:getBooksFromPeriod(period_begin, period_end, callback_
         table.insert(results, {
             result_book[1][i],
             T(_("%1 (%2)"), util.secondsToClockDuration(user_duration_format, tonumber(result_book[2][i]), false), tonumber(result_book[3][i])),
+            book_id = tonumber(result_book[4][i]),
```

Comment:
Why was that needed ? Or it is now with your added code ?

Reply:
I need this to pair book with the reading periods that I got from the other query, otherwise only title can be used and there might be duplicate titles for different book entries.

Reply:
Oups, my mind had read that single line as:
```diff
-book_id = result_book[4][i],
+book_id = tonumber(result_book[4][i]),
```
so my questions were about this imaginary diff :)